cmd/internal/cli/pull.go

```diff
@@ -270,7 +270,7 @@ func pullRun(cmd *cobra.Command, args []string) {
 
 		// check if we pulled from the library, if so; is it signed?
 		if !unauthenticatedPull {
-			imageSigned, err := signing.IsSigned(name, KeyServerURL, 0, false, authToken, true)
+			imageSigned, err := signing.IsSigned(name, KeyServerURL, 0, false, authToken, false, true)
```

Comment:
boolean arguments are usually a hint of a function doing multiple things, as in:

```
func something(a, b, c bool) {
    if a {
        // do "a"
    }
    if b {
        // do "b"
    }
    if c {
        // do "c"
    }
}
```

I haven't looked at the change in the function, but would it be possible to add another function with a specific purpose?